crowdin/source_files.go

```diff
@@ -0,0 +1,288 @@
+package crowdin
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/crowdin/crowdin-api-client-go/crowdin/model"
+)
+
+// Source files are resources for translation. You can keep files structure using
+// folders or manage different versions of the content via branches.
+// Use API to keep the source files up to date, check on file revisions, and manage project branches.
+// Before adding source files to the project, upload each file to the Storage first.
+//
+// Note: If you use branches, make sure your master branch is the first one you integrate with Crowdin.
+//
+// Crowdin API docs: https://developer.crowdin.com/api/v2/#tag/Source-Files
+type SourceFilesService struct {
+	client *Client
+}
+
+// ListBranches returns a list of project branches.
```

Comment:
Also include the String-based related branch methods in the `source_files` section.

https://developer.crowdin.com/api/v2/string-based/#tag/Branches

Reply:
Is it possible to move all brunch-related methods to a separate service (e.g. Branches)?

Reply:
Let's move it to a separate service